components/brave_component_updater/browser/BUILD.gn

```diff
@@ -1,5 +1,17 @@
 import("//extensions/buildflags/buildflags.gni")
 
+declare_args() {
+  updater_dev_endpoint = ""
+  updater_prod_endpoint = ""
+}
+
+config("network_config") {
```

Comment:
nit - this name doesn't really make sense to me, I could call it either `updater_config` or just `config` and make it a public config for `browser` as mentioned below